test/verify/check-system-info

```diff
@@ -566,26 +566,28 @@ revision\t: 2.3 (pvr 004e 1203)
             b.wait_in_text('#hwinfo #memory-listing table', "DIMM")
             b.wait_in_text('#hwinfo #memory-listing table', "RAM")
 
-            # Test more specific memory data with a fake dmidecode
-            m.write('/tmp/dmidecode', dmidecode)
-            m.execute('chmod +x /tmp/dmidecode; mount -o bind /tmp/dmidecode $(which dmidecode)')
-            self.addCleanup(m.execute, 'until umount $(which dmidecode); do sleep 1; done')
-            b.reload()
-            b.enter_page('/system/hwinfo')
-            # first DIMM has unknown technology and rank
-            b.wait_in_text('#memory-listing tr:nth-of-type(1) td[data-label=ID]', "BANK 0: ChannelA-DIMM0")
-            b.wait_in_text('#memory-listing tr:nth-of-type(1) td[data-label=Type]', "DDR3")
-            b.wait_in_text('#memory-listing tr:nth-of-type(1) td[data-label=Size]', "8 GiB")
-            b.wait_in_text('#memory-listing tr:nth-of-type(1) td[data-label=State]', "Present")
-            b.wait_text('#memory-listing tr:nth-of-type(1) td[data-label="Memory technology"]', "Unknown")
-            b.wait_text('#memory-listing tr:nth-of-type(1) td[data-label=Rank]', "Unknown")
-            b.wait_in_text('#memory-listing tr:nth-of-type(1) td[data-label=Speed]', "1600 MT/s")
-            # second DIMM has known technology and rank
-            b.wait_in_text('#memory-listing tr:nth-of-type(2) td[data-label=ID]', "BANK 0: ChannelB-DIMM0")
-            b.wait_text('#memory-listing tr:nth-of-type(2) td[data-label="Memory technology"]', "Awesome")
-            b.wait_text('#memory-listing tr:nth-of-type(2) td[data-label=Rank]', "Single rank")
-            # Locator can be the same on multi-channel memory platforms - bank locator should differentiate them
-            b.wait_in_text('#memory-listing tr:nth-of-type(3) td[data-label=ID]', "BANK 1: ChannelB-DIMM0")
+            systemd_version = int(m.execute("systemctl --version | awk '{print $2; exit}'").strip())
+            if systemd_version < 248:
```

Comment:
Can you please add an equivalent `udevadm` mock, to have the parsing of that in CI? Ideally the assertions should be exactly the same, just the mock setup should differ.